### PR TITLE
Meld

### PR DESCRIPTION
**Pull request by maxbruening**
Add basic support for the Meld diff editor on windows.

### CODE REVIEW

File: apps/meld/meld.talon [hunk 1]
```diff
@@ -0,0 +1,6 @@
+app: meld
+-
+tag(): user.tabs
+
+change next: key(alt-up)
```
Comment (knausj85) [inline apps/meld/meld.talon:5]: I believe these shortcuts are swapped

File: apps/meld/meld.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    def tab_reopen():
+        print("Meld does not support this action.")
+
+
+@ctx.action_class("user")
+class UserActions:
+    def tab_jump(number):
+        if number < 10:
+            actions.key(f"alt-{number}")
```
Comment (knausj85) [inline apps/meld/meld.py:38]: This doesn't appear to work, though the shortcut appears to be correct

Comment (maxbruening) [reply apps/meld/meld.py:38]: It works for me on windows 11, But it seems like Meld implements this keyboard shortcut in an unfortunate way:  it seems to work only if the current tab is selected, for example by clicking on the current tab.

Comment (maxbruening) [reply apps/meld/meld.py:38]: See here [https://gitlab.gnome.org/GNOME/meld/-/issues/839#note_2077059]()

**Review by knausj85 — APPROVED** [review]

**Pull request merged**